src/api/firebase.js

```diff
@@ -122,3 +122,24 @@ export async function deleteItem() {
 	 * this function must accept!
 	 */
 }
+
+export function comparePurchaseUrgency(data) {
```

Comment:
This function is way drier than the previous code--great job on the refactor! I think it can potentially be taken even further by calling a filter on the data only once and sorting it into an active or inactive empty array. 
`let activeItems = []
let inactiveItems = []

	sortItems(data.filter((item) => {
		return (
			item.dateLastPurchased === null || getItemDaysUntilNextPurchase(item) <= 60 ? activeItems.push(item) : inactiveItems.push(item)
		);
	}))`